src/components/media-player/media-player.js

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect } from "react";
+import uniqueId from "lodash/uniqueId";
+
+import { VideoPlayer, AcceptOverlay } from "bootstrap-italia";
+import parse from "html-react-parser";
+import ReactMarkdown from "react-markdown";
+
+// TO DO
+// - [future] modular approach for local videos/youtubes/...
+// - [future] download thumbnail during build - eg. url: http://i3.ytimg.com/vi/_0j7ZQ67KtY/hqdefault.jpg
+
+function MediaPlayerEl({ url, lang, trascription, poster }) {
+  const messages = {
+    it: {
+      rememberLabel: "Ricorda per tutti i video",
+      acceptLabel: "Accetta",
+      trascriptionLabel: "Trascrizione",
+      cookiePolicy:
+        'Accetta i cookie di YouTube per vedere il video. Puoi gestire le preferenze nella <a href="/privacy-policy/#gestione-cookie" className="text-white">cookie policy</a>. ',
+    },
+    en: {
+      rememberLabel: "Remember my choice",
+      acceptLabel: "Accept",
+      trascriptionLabel: "Transcription",
+      cookiePolicy:
+        'You have to accept YouTube cookies to enable viewing of this video. You can manage the preferences via <a href="/privacy-policy/#gestione-cookie" className="text-white">cookie policy (italian language)</a>.',
+    },
+  };
+
+  let video = null;
+  const t = (key) => messages[lang][key];
+
+  const videoId = `video-js-${uniqueId("id_")}`;
+
+  useEffect(() => {
+    const script = document.createElement("script");
+    script.src =
+      "https://cdnjs.cloudflare.com/ajax/libs/videojs-youtube/3.0.1/Youtube.min.js";
```

Comment:
Can we host this script ourselves? Ideally using the npm package, so the dependency is declared in the project

Reply:
WDYT @astagi ?

Reply:
Note: just as a reminder: we embed this also on the BSI example template `base.html`.

Reply:
See https://github.com/italia/designers.italia.it/pull/756#discussion_r1355322427

Reply:
@Fupete this is different than making requests to YouTube